libs/deal-ticket/src/hooks/use-fee-deal-ticket-details.tsx

```diff
@@ -53,19 +53,22 @@ export interface FeeDetails {
 const emptyValue = '-';
 const formatValue = (
   value: string | number | null | undefined,
-  formatDecimals: number
+  formatDecimals: number,
+  quantum?: string
 ): string => {
+  const numberDp = Math.max(0, Math.log10(100 / Number(quantum) || 0));
   return isNumeric(value)
-    ? addDecimalsFormatNumber(value, formatDecimals)
+    ? addDecimalsFormatNumber(value, formatDecimals, numberDp)
```

Comment:
Most probably we will be using this function also in other places so it will be useful to have this in `libs/utils/sec/lib/format/number.ts` as something like this

```
export const addDecimalsFormatNumberQuantum = (
  rawValue: string | number,
  decimalPlaces: number,
  quantum: number | string
) =>
  addDecimalsFormatNumber(
    rawValue,
    decimalPlaces,
    Math.max(0, Math.log10(100 / Number(quantum) || 0))
  );
```

Reply:
added